extensions/ql-vscode/src/common/discovery.ts

```diff
@@ -1,5 +1,5 @@
 import { DisposableObject } from "../pure/disposable-object";
-import { extLogger } from "../common";
+import { extLogger } from ".";
```

Comment:
Let's try and avoid this since we're already within the same module. Otherwise following the dependencies gets a bit circular.

```suggestion
import { extLogger } from "./logging/logger";
```

Warning: browser coding!

Reply:
Thanks! Makes sense 🧠